package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "helpshift-docs",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
```

Comment:
What does clear do? Is it the same as the "clean" task in our Makefile?

Reply:
This is a CLI command given to us by Docusaurus (https://docusaurus.io/docs/cli#docusaurus-clear-sitedir). As far as I know, it gets rid of the local `.docusaurus` folder. It's not similar to the clean task in our Makefile because I don't think it gets rid of node_modules.

Reply:
Ack